### ISSUE TITLE
Swift 5.3 error

### ISSUE THREAD

**Issue by jcleigh** [bringg/ReactNativeBringgDriverSDK#14]
We get this when building in XCode 12:

```
Module compiled with Swift 5.2.4 cannot be imported by the Swift 5.3 compiler: <redacted path>/ios/Pods/BringgDriverSDK/BringgDriverSDK.framework/Modules/BringgDriverSDK.swiftmodule/x86_64.swiftmodule
```

Tried removing `node_modules` and `Pods` and downloading fresh, no difference.

**Comment by jcleigh** [comment]
Update: I removed the Bringg SDK from `package.json` and `Podfile`, reinstalled dependencies, and the app built. That indicates that it is clearly the Bringg SDK compiled Swift version causing the issue.

**Comment by jbrad** [comment]
Any updates on this issue?

**Comment by Bryanjfh** [comment]
Yes yesterday I was trying XCode 12 with the new bringg SDK version. Good news is that everything seems good with :bringg: bad news is that for XCode 12/iOS 14, images are not being rendered and we need to update to react-native >= 0.63.2 and by doing so I was fighting a lot of dependencies issues

**Comment by jcleigh** [comment]
XCode 12.0 and 12.1 are working, but now we see it with 12.2:
```Module compiled with Swift 5.3 cannot be imported by the Swift 5.3.1. compiler```

**Comment by jbrad** [comment]
I'm seeing the same thing with XCode 12.2
